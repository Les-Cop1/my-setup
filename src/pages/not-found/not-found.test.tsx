import React from 'react'

import { NotFound } from '@pages'
import '@testing-library/jest-dom'
import { render } from '@testing-library/react'

describe('Not found', () => {
  test('Snapshot', () => {
    const { container } = render(<NotFound />)
    expect(container).toMatchSnapshot()
  })
})
