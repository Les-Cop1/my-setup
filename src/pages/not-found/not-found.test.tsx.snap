// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not found Snapshot 1`] = `
<div>
  <div
    class="w-full flex flex-col items-center justify-center p-10 gap-6"
  >
    <img
      alt="page not found illustration"
      class="max-h-96"
      src=""
    />
    <h1>
      This page doesn't exist
    </h1>
  </div>
</div>
`;
